lib/VueFinalModal.vue

```diff
@@ -323,8 +323,17 @@ export default {
       this.$emit('click-outside', this.createModalEvent({ type: 'click-outside' }))
       this.clickToClose && this.$emit('input', false)
     },
-    onEsc(evt) {
-      if (evt.keyCode === 27 && this.visible && this.escToClose) {
+    onKeyDown(evt) {
+      switch (evt.keyCode) {
+        case 27:
+          this.onEsc()
+          break
+        default:
+          return false
+      }
+    },
```

Comment:
Hi! @fatihsolhan

Thank your PR. But I want to understand, why you want to use one more function and `switch/case` for processing ?

And the other, how about use lookup table instead of `switch/case`, I think that have lower cyclomatic complexity and has better scalability.

like below:
```js
onKeyDown (evt) {
  const keyEventMap = {
    27: this.onEsc
  }
  
  (keyEventMap[evt.keyCode] || (() => false))()
}
```

Reply:
Awesome! I also think `@keydown.esc="onEsc"` is better.

Thx you 🎉